build-config/src/main/resources/weld-checkstyle/checkstyle.xml

```diff
@@ -50,9 +50,9 @@
             <property name="ignoreNumbers" value="-1, 0, 1, 2, 3"/>
         </module>
         <module name="MissingSwitchDefault" />
-        <module name="RedundantThrows">
+        <!--<module name="RedundantThrows">
```

Comment:
Didn't occur to me to check the commit msg for that info :)
Well in that case I believe we can cross out `RedundantThrows` entirely - no point in keeping it there if it was removed.
As for `RedundantModifier`, there aren't exactly many ways to [configure](http://checkstyle.sourceforge.net/config_modifier.html#RedundantModifier) it. I will check how much work would it be to correct the code.